src/runtimes/node/index.js

```diff
@@ -4,14 +4,29 @@ const cpFile = require('cp-file')
 
 const { JS_BUNDLER_ESBUILD, JS_BUNDLER_ESBUILD_ZISI, JS_BUNDLER_ZISI, RUNTIME_JS } = require('../../utils/consts')
 
+const { detectEsModule } = require('./detect_es_module')
 const { findFunctionsInPaths } = require('./finder')
 const { getSrcFiles } = require('./src_files')
 const { zipEsbuild } = require('./zip_esbuild')
 const { zipZisi } = require('./zip_zisi')
 
 // We use ZISI as the default bundler until the next major release, with the
 // exception of TypeScript files, for which the only option is esbuild.
-const getDefaultBundler = ({ extension }) => (extension === '.ts' ? JS_BUNDLER_ESBUILD : JS_BUNDLER_ZISI)
+const getDefaultBundler = ({ extension, srcFile }) => {
+  if (extension === '.ts') {
+    return JS_BUNDLER_ESBUILD
+  }
+
+  if (srcFile) {
```

Comment:
Do we need this check here, since `zipFunction` is the only function calling this and `join()` will never be falsy?